src/main/java/teammates/common/util/Const.java

```diff
@@ -1362,6 +1376,14 @@ private Const() {
         public static final String FEEDBACK_SESSION_PUBLISHED =
                 "The feedback session has been published. "
                 + "Please allow up to 1 hour for all the notification emails to be sent out.";
+        public static final String FEEDBACK_SESSION_NOTPUBLISHED =
+                "The feedback session is not published. "
+                + "You cannot send publication reminders for a session that is not published.";
+        public static final String FEEDBACK_SESSION_PUBLISHEDEMAILEMPTYRECIPIENT =
```

Comment:
May be can add underscore between each word as they are quite hard to read

Reply:
Here I'm following `FEEDBACK_SESSION_REMINDERSSESSIONNOTOPEN` and `FEEDBACK_SESSION_REMINDERSEMPTYRECIPIENT`, `FEEDBACK_QUESTION_NUMBEROFENTITIESINVALID`

Reply:
But it seems the rest are all separated by the underscore, what about others' opinion? @whipermr5

Reply:
I guess can underscore. Btw, use a prefix e.g. `FEEDBACK_SESSION_RESEND` to group constants related to this new feature together.

Reply:
Ok 👍 then how about other constants like `FEEDBACK_SESSION_REMINDERSSESSIONNOTOPEN`, `FEEDBACK_QUESTION_NUMBEROFENTITIESINVALID`, etc?

Reply:
Don't need to fix in this PR as it's out of the scope?